### ISSUE TITLE
Possible integration issue with Spring Security Login form

### ISSUE THREAD

**Issue by PritiShah** [formvalidation/support#197]
Hello, 

Just wanted to get to your attention a possible integration issue with Spring Security Login form.

I have bought the latest version of bootstrap validator - formValidator, few days back. I have been trying to incorporate this validator with my existing framework of Spring security specifically the spring security login form. Now this form needs its submit button to be named as "submit" and for this reason I cannot get the formValidator to work as expected. Is there any work around to this issue?

Thanks,
Priti

**Comment by nghuuphuoc** [comment]
> Now this form needs its submit button to be named as "submit"

In Spring, does the submit button's name have to be `submit`?
Because it seems that we can change that 
* http://docs.spring.io/spring-security/site/docs/3.2.x/guides/form.html
* http://www.mkyong.com/spring-security/spring-security-form-login-example/

Not only FormValidation, but also usual form suggests that we should avoid this name.
See http://formvalidation.io/examples/cant-submit-form-after-validation/#the-submit-buttons-name-is-not-valid

**Comment by nghuuphuoc** [comment]
Hi @PritiShah 
Can you change the name of submit button as I suggested?
Is the problem solved?

**Comment by PritiShah** [comment]
Hello,

1st link is not exactly relevant since its simple Spring form. 2nd link still has the name of submit button as "submit". This link http://www.javacodegeeks.com/2013/05/spring-security-login.html
and many other relevant links  has the submit button named submit. May be Spring security demo 
examples might have it named as "submit". 

Initially I did think that "submit" name might be the problem since formValidation didnt happen if I kept btn name as "submit" but spring security worked properly and if I removed name as "submit" spring security didnt work as expected but formValidation event occured!  But after reading your response, I think my error was in the part of calling formValidation in a manual way, like below in my js file as opposed to your solution of - on('success.form.fv', function(e) {....}.

var isFormValid = $('#' + form).data('formValidation').validate().isValid();
	if (isFormValid){
             continue method ......
        }

Nevertheless, I removed the name = "submit" and the formValidation part and it was working ok. So, good news - it doesnt have to be named "submit" for spring security. 

However, I had done manual validation for other functions too and all seem to work well except for this one. It might be due to extra time taken by spring security to process the login. The error I kept getting was -  Uncaught TypeError: $(...).formValidation is not a function .

Thanks a lot! 

Priti
